OverlayPlugin.Core/MemoryProcessors/Party/PartyMemory.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.Party
+{
+    public class PartyListEntry
+    {
+        public float x;
+        public float y;
+        public float z;
+        public long contentID;
+        public uint objectID;
+        public uint currentHP;
+        public uint maxHP;
+        public ushort currentMP;
+        public ushort maxMP;
+        public ushort territoryType;
+        public ushort homeWorld;
+        public string name;
+        public byte sex;
+        public byte classJob;
+        public byte level;
+        public byte flags;
+    }
+
+    public class PartyListsStruct
+    {
+        public long partyId;
+        public long partyId_2;
+        public uint partyLeaderIndex;
+        public byte memberCount;
+        public byte allianceFlags;
+
+        public uint Unk_3D40;
+
+        public PartyListEntry[] partyMembers;
+        public PartyListEntry[] allianceAMembers;
+        public PartyListEntry[] allianceBMembers;
+        public PartyListEntry[] allianceCMembers;
+        public PartyListEntry[] allianceDMembers;
+        public PartyListEntry[] allianceEMembers;
```

Comment:
Are you missing F here? Alternatively, maybe if we don't know how to determine D/E/F (since from your other comments, we won't get party updates for out of instance cross-world parties, which is the only place D/E/F occurs) we should just drop all of these members and checks?

Reply:
In reality which alliance these point to is dependent on what alliance the player is currently in.

Assume the player is in alliance B, then:
`partyMembers` contains alliance B.
`allianceAMembers` contains alliance A.
`allianceBMembers` contains alliance C.

But if the player is in alliance C, then `A` contains `A`, and `B` contains `B`.

I couldn't come up with a better way to name them.

As for D/E/F, those might still happen in-instance, I'm not sure. The structure exists for some reason at least.

Reply:
Aaaaaaaa.  Can these be numbered instead of letters then, so there's less confusion?